internal/ui/context.go

```diff
@@ -342,6 +343,14 @@ func (g *globalState) setScreenFilterEnabled(enabled bool) {
 	atomic.StoreInt32(&g.screenFilterEnabled_, v)
 }
 
+func (g *globalState) storeGraphicsLibrary(library GraphicsLibrary) {
```

Comment:
Use the name `graphicsLibrary` as the other functions for consistency. (This might be odd but I'd want to keep this consistent)

Reply:
Do you mean on this one, or on the `loadGraphicsLibrary`?

Reply:
Ah, sorry, rename `loadGraphicsLibrary` -> `graphicsLibrary` and `storeGraphicsLibrary` -> `setGraphicsLibrary`